Parse/Parse.h

```diff
@@ -81,7 +81,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param applicationId The application id of your Parse application.
  @param clientKey The client key of your Parse application.
  */
-+ (void)setApplicationId:(NSString *)applicationId clientKey:(NSString *)clientKey;
++ (void)setApplicationId:(NSString *)applicationId clientKey:(nullable NSString *)clientKey;
```

Comment:
This should still be non null. If the server URL isn't changed (which only happens via the configuration initialization) client key is still required.